### ISSUE TITLE
Provide an iframe embed option that has an accessible title attribute

### ISSUE THREAD

**Issue by vijay-howlround** [JohnXLivingston/peertube-plugin-livechat#110]
Thank you for this excellent project. We are making good use of the livechat plugin for PeerTube. 

**Is your feature request related to a problem? Please describe.**
We frequently embed two iframes on our website when we publish live stream events: one for the live video player, and one for the live chat. From the PeerTube page, we click the "Share > Embed" and then paste the video player iframe code to our webpage. When embedding the livechat, we click on the "Share chat link" button, and then we manually edit an iframe from the chat URL that is provided. Having to manually write an iframe is difficult for us, and sometimes prone to error.       

**Describe the solution you'd like**
When we click "Share chat link", we would like simply to copy an automatically proposed iframe code. Since we are aiming to comply with WCAG accessibility standards and distinguish the live video player iframe from the live chat iframe (for the benefit of people using assistive technologies), we would like the iframe code to include a specific Title Attribute with the following syntax: title="web chat for [insert live event's Title]".

**Additional context**
This may be related to issue #92, but I am not sure.

**Comment by JohnXLivingston** [comment]
Thanks for your request @vijay-howlround . 

Good news: I already planned to do so, and I recently found a funding to do this development. It should be available very soon (by the end of this year).
There should also be an option to open the chat in the video iframe. Recent Peertube version make it possible, and it was asked in the issue you mentioned. I will offer both solutions: chat in the video iframe, or in a dedicated iframe. Because I think the 2 use case are usefull.

For the title attribute, I did not think about it. Thanks for the idea!

**Comment by vijay-howlround** [comment]
> I will offer both solutions: chat in the video iframe, or in a dedicated iframe. Because I think the 2 use case are useful. For the title attribute, I did not think about it. Thanks for the idea!

That will be great to have those two iframe options available, and a descriptive title attribute for accessibility. Thank you!

**Comment by JohnXLivingston** [comment]
@vijay-howlround , here we go! Will be available in v6.1.0 of the plugin.

By clicking on the «share button» above the chat:

![image](https://user-images.githubusercontent.com/38844060/211884392-6c90bc70-e396-4aef-981f-55f8b89866af.png)

This generate the following iframe:

```html
<iframe src="http://localhost:9000/plugins/livechat/router/webchat/room/40d4101d-51e8-4b6c-9a5d-1cfd129def2b" title="Chat for live stream: Live test" sandbox="allow-same-origin allow-scripts allow-popups allow-forms" width="560" height="315" frameborder="0"></iframe>
```

The title will be «Chat for live stream: the video title».
For now, I only have the french and english for «Chat for live stream:» (other translations are welcome, if there is any translator here :) ).

And the result:
![image](https://user-images.githubusercontent.com/38844060/211885139-063591b0-4d95-42a0-8e99-d2e1b9809234.png)

Is this OK for you?
Are the iframe attributes OK for you?

**Comment by JohnXLivingston** [comment]
I'm closing the issue, as it is implemented by https://github.com/JohnXLivingston/peertube-plugin-livechat/commit/ce9d531db77f4bc68b9d0f08fed3b520f1677a92
Feel free to re-open if there is any problem.

**Comment by vijay-howlround** [comment]
@JohnXLivingston This is great! Yes, the title attribute is helpful. Thank you so much!

**Comment by JohnXLivingston** [comment]
You are welcome :)
And thanks Framasoft for funding this feature! (and also #92 , which is not done yet)

I don't have a release date, I have some other works to do before. Should be this month.

**Comment by JohnXLivingston** [comment]
@vijay-howlround , I just released the v6.1.0. Sorry for the delay.

If you don't want to wait for your instance to index the new version (which can last 12 or 24 hours), you can do this modification in the database:

```
sudo -u peertube psql peertube_prod
peertube_prod=> update "plugin" set "latestVersion" = '6.1.0' where "plugin"."name" = 'livechat';
UPDATE 1
```

**Comment by vijay-howlround** [comment]
@JohnXLivingston Fantastic. Thank you so much!
